db/seeds/source_types.yml

```diff
@@ -123,6 +123,32 @@ ansible-tower:
         :condition:
           :when: endpoint.verify_ssl
           :is: true
+google:
+  :product_name: Google Cloud Engine
+  :vendor: Google
+  :icon_url: "/apps/chrome/assets/images/partners-icons/google-cloud-engine.svg" # TODO
+  :schema:
+    :authentication:
+    - :type: username_password
+      :name: Project ID and Service Account JSON
```

Comment:
is the Service Account JSON going to be used to seed the service account email for the wizard?

Reply:
I don't know, but in the credentials provided by QE there is a `client_email` key-value in the JSON

Reply:
Anyway, I started from the ManageIQ's create form: https://files.gitter.im/5aa15e4ad73408ce4f907617/dsFs/Screenshot-2021-01-15-at-10.44.16.png

Reply:
Are there other use cases for GCP that you are working on besides Cost Management?

Reply:
There is and oportunity for topological-inventory collector, but we don't have any milestone for that as far as i know